tcms/kiwi_auth/tests/test_admin.py

```diff
@@ -188,3 +191,33 @@ def test_should_be_able_to_edit_a_non_default_group(self):
             '<input type="text" name="name" value="%s" class="vTextField"'
             ' maxlength="150" required id="id_name">' % self.group.name,
         )
+
+    def test_should_be_allowed_to_create_new_group_with_added_user(self):
+        group_name = "TestGroupName"
+        response = self.client.post(
+            reverse("admin:auth_group_add"),
+            {"name": group_name, "users": [self.tester.id]},
+            follow=True,
+        )
+
+        self.assertContains(response, group_name)
+        self.assertContains(response, "was added successfully.")
+        self.assertTrue(self.tester.groups.filter(name=group_name).exists())
+
+    def test_should_be_able_to_add_user_while_editing_a_group(self):
+        self.client.post(
+            reverse("admin:auth_group_change", args=[self.group.id]),
+            {"name": self.group.name, "users": [self.tester.id]},
+            follow=True,
+        )
+
+        response = self.client.get(
+            reverse("admin:auth_group_change", args=[self.group.id])
+        )
```

Comment:
If you post `submit: _continue` I believe this will emulate clicking on "Save & continue" and b/c you follow the redirects the response will be again the change page. So you can avoid fetching the page one more time.